src/BitStringAddresses/fockaddress.jl

```diff
@@ -209,35 +209,40 @@ See [`SingleComponentFockAddress`](@ref).
 """
 excitation
 
+
 """
-    OccupiedModeMap(addr) <: AbstractVector
+    ModeMap(addr) <: AbstractVector
```

Comment:
```suggestion
    ModeMap <: AbstractVector
```
We are only documenting a type, not a constructor. Hence we should leave out the `()`, which would indicate that you can use it as a function/constructor.